Formula/gmp.rb

```diff
@@ -25,12 +25,11 @@ def install
       args << "ABI=32"
     end
 
-    # https://github.com/Homebrew/homebrew/issues/20693
-    args << "--disable-assembly" if build.build_32_bit? || build.bottle?
+    args << "--enable-fake-cpuid" << "GMP_CPU_TYPE=#{Hardware.oldest_cpu}" << "--enable-fat" if build.build_32_bit? || build.bottle?
 
     system "./configure", *args
     system "make"
-    system "make", "check"
+    system "make", "check", "GMP_CPU_TYPE=#{Hardware.oldest_cpu}"
```

Comment:
Could this perhaps be set in both places with `ENV["GMP_CPU_TYPE"] = Hardware.oldest_cpu`?